README.md

```diff
@@ -12,6 +12,7 @@ A simple electronic punch clock to track hours spent on projects.
 Ruby 2.7.5
 Rails 6.0.3
 Postgres >= 9.1
+Node 16.13.1
```

Comment:
Had some issues with installing.
Although not related to the PR, I think it will be better to specify the Node version.

Reply:
nice, the package.json should already handle this but is nice to also have here, just need to pay attention to keep updated